src/list.areas.json

```diff
@@ -198,5 +198,37 @@
                 "accept-language": "en"
             }
         }
+    },
+    "spain-autonomous-communities": {
+        "name": "Spain autonomous communities",
+        "description": "",
+        "author": "Rukant",
+        "data": {
+            "urlArea": "https://raw.githubusercontent.com/GeoGuess/GeoGuess-Maps/main/public/geojson/areas/spain_autonomous_communities.geojson",
+            "type": "nominatim",
+            "pathKey": "NAME_1",
+            "nominatimResultPath": "address.state",
+            "nominatimQueryParams": {
+                "zoom": "5",
+                "addressdetails": "1",
+                "accept-language": "es"
+            }
+        }
+    },
+    "spain-provinces": {
```

Comment:
Unfortunately, we have an issue in this geojson, some "provincia" values don't match with OpenStreetMap :
1. Received from openstreetmap "Valencia" but "València"  in geojson
Tested with https://nominatim.openstreetmap.org/reverse?format=json&lat=39.4271294470365&lon=-0.8663281562502531&zoom=8&addressdetails=1&accept-language=es
2. Received from openstreetmap "Gerona" but "Girona" in geojson
Tested with https://nominatim.openstreetmap.org/reverse?format=json&lat=42.124818507961336&lon=2.729617274154627&zoom=8&addressdetails=1&accept-language=es
2. Received from openstreetmap "Alicante" but "Alacant" in geojson
Tested with https://nominatim.openstreetmap.org/reverse?format=json&lat=38.52134644552799&lon=-0.37103329879899116&zoom=8&addressdetails=1&accept-language=es
2. Received from openstreetmap "Región de Murci" but "Murcia" in geojson
Tested with https://nominatim.openstreetmap.org/reverse?format=json&lat=37.82459302432988&lon=-1.3477247855292498&zoom=8&addressdetails=1&accept-language=es
2. Received from openstreetmap "Lérida" but "Lleida" in geojson
Tested with https://nominatim.openstreetmap.org/reverse?format=json&lat=42.06838324616863&lon=1.1442427212390878&zoom=8&addressdetails=1&accept-language=es